deepparse/cli/parser_arguments_adder.py

```diff
@@ -1,13 +1,13 @@
 from argparse import ArgumentParser
 
 from .tools import wrap, bool_parse
-from .. import MODEL_CHOICES
+from .. import MODEL_MAPPING_CHOICES
 
 
 def add_base_parsing_model_arg(parser: ArgumentParser) -> None:
     parser.add_argument(
         "base_parsing_model",
-        choices=MODEL_CHOICES,
+        choices=list(MODEL_MAPPING_CHOICES),
```

Comment:
Pourquoi ici un list(...) ?

Reply:
Since MODEL_MAPPING_CHOICES is now a Dict I wanted to keep the choices arg as a list.

Reply:
And what about the other case? It is the only time you use a List.

Reply:
Looked more closely in how choices is handled in add_argument, it verifies with the values with the `_check_value` method which uses __contain__ (in) to verify if the arg is valid or not. Since this method  implemented in a dictionary it still works.